kittens.css

```diff
@@ -1,24 +1,25 @@
 body {
-    background: black;
+    background: #0C2026;
 
 }
 
 section {
-    background: white;
     width: 1100px;
     margin: 0 auto;
 }
 
 h1 {
-    background: blue;
+    padding-top: 40px;
+    background: #757767;
     width: 1000px;
-    height: 100px;
+    height: 80px;
```

Comment:
Just a note that most code linters (error checkers) will tell you that you should at least be careful restricting any elements by both height and width.